pkg/pr/prupdate_test.go

```diff
@@ -44,6 +44,7 @@ func TestExecuteUpdate(t *testing.T) {
 			expectedErr:      nil,
 			modifiedFiles:    "pkg/cmd/lint.go\npkg/lint/lint.go\n",
 			existingBranches: "main\ndifferentBranch\n",
+			currentChanges:   "M  pkg/cmd/lint.go\n M  pkg/lint/lint.go\n",
```

Comment:
```suggestion
			currentChanges:   "M  pkg/cmd/lint.go\nM  pkg/lint/lint.go\n",
```
I don't think you will get that extra space before the M on the second line if it has two spaces after